bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -9,7 +9,8 @@ metadata:
     heritage: {{ .Release.Service | quote }}
 spec:
   serviceName: {{ template "zookeeper.fullname" . }}-headless
-  replicas: 1
+  replicas: {{ .Values.replicaCount }}
+  podManagementPolicy: {{ .Values.podManagementPolicy | quote }}
   updateStrategy:
```

Comment:
Can you apply the changes I requested on this PR on Redis? https://github.com/kubernetes/charts/pull/6121

As you can check, I asked for a parameters to indicate the partitions when using **RollingUpdate**